File: db/transport_localized.php
<?php

require_once("entity.php");

/**
 * Class for accessing, manipulating, creating and deleting records in transport_localized.
*/

class Transport_Localized extends Entity
{

  public function __construct($select = array())
  {
    $this->table = "transport_localized";
    $this->order = "";
    $this->domain = "localization";
    $this->field['transport_id']['type'] = 'INTEGER';
    $this->field['transport_id']['not_null'] = true;
    $this->field['language_id']['type'] = 'INTEGER';
    $this->field['language_id']['not_null'] = true;
    $this->field['name']['type'] = 'VARCHAR';
    $this->field['name']['length'] = 64;
    $this->field['name']['not_null'] = true;
    $this->field['transport_id']['primary_key'] = true;
    $this->field['language_id']['primary_key'] = true;
    parent::__construct($select);
  }

}

?>

File: pentabarf/lib/momomoto/tables/transport_localized.rb
module Momomoto
  class Transport_localized < Base
    def initialize
      super
      @domain = 'localization'
      @fields = {
        :transport_id => Datatype::Integer.new( {:not_null=>true, :primary_key=>true} ),
        :language_id => Datatype::Integer.new( {:not_null=>true, :primary_key=>true} ),
        :name => Datatype::Varchar.new( {:not_null=>true, :length=>64} )
      }
    end
  end
end
